securedrop/tests/functional/app_navigators.py

```diff
@@ -0,0 +1,252 @@
+import time
```

Comment:
Most of this is existing code taken from `FunctionalTest` but "repackaged" as separate, smaller/simpler classes.